packages/calypso-e2e/src/lib/components/navbar-component.ts

```diff
@@ -2,7 +2,7 @@ import { Page } from 'playwright';
 
 const selectors = {
 	// Buttons on navbar
-	mySiteButton: 'button[data-tip-target="my-sites"]',
+	mySiteButton: '[data-tip-target="my-sites"]',
```

Comment:
On most of Calypso, this is indeed a button.

However, after launching the site, this becomes a hyperlink instead.